### DISCUSSION TITLE
CHIRPS: Rainfall Estimates from Rain Gauge and Satellite Observations

### DISCUSSION THREAD

**Discussion by TomAugspurger** [microsoft/PlanetaryComputer#106] [Data request]
Homepage: https://chc.ucsb.edu/data/chirps
Data download: https://data.chc.ucsb.edu/products/CHIRPS-2.0/
Temporal Frequency: Daily, Monthly
Temporal Extent: 1981 - Present
Spatial Extent: quasi-Global + Regional
Spatial Resolution: 0.05 degree
File format: TIF (seemingly COGs)
Other notes:
- Two products: rapid and final

**Comment by KBodolai** [comment]
It would be great to have a CHIRPS dataset in the planetary computer, there's [version 3](https://data.chc.ucsb.edu/products/CHIRPS/v3.0/pentads/global/) now, with each year of data being around 1GB in size.
